python/lsst/pipe/tasks/multiBand.py

```diff
@@ -697,6 +697,19 @@ class MergeMeasurementsConfig(MergeSourcesConfig):
     pseudoFilterList = ListField(dtype=str, default=["sky"],
                                  doc="Names of filters which may have no associated detection\n"
                                      "(N.b. should include MergeDetectionsConfig.skyFilterName)")
+    snName = Field(dtype=str, default="base_PsfFlux",
+                       doc="Name of flux measurement for calculating the S/N when choosing the "
+                           "reference band.")
```

Comment:
We can justify the choice of PSF flux because it is in practice the most robust.  I justified the minSN number for HSC based on the data, but the minSNDiff is completely unjustified.  I don't know if this information should be recorded somewhere, but it would be useful to look at these numbers again more generally.  I'll create a ticket for them.

Fixed the indentation.